pants.toml

```diff
@@ -96,8 +96,8 @@ root_patterns = [
   "/contrib/examples/actions/ubuntu_pkg_info", # python script runs via shell expecting cwd in PYTHONPATH
   # lint plugins
   "/pylint_plugins",
-  # pants plugins
-  "/pants-plugins",
+  # pants plugins - 202404 disabled because pants doesn't support >py3.9 and conflicts with other targets using >py3.9
+#  "/pants-plugins",
```

Comment:
What issue(s) led to commenting this?

I believe this is what we need to change to add py3.10 to the interpreter constraints: 
753974dd56db075017f947d287c6cb957747e10d

(That commit is in the pants-ic-py3.10 branch which includes the update-oslo.config branch)

Reply:
On ubuntu 22.04 there is no python3.9 interpreter and no matter if I tried py3.10 or py3.11 or bump pants to 2.18 it refused to run because it claimed to need py3.9.  I couldn't figure out how to find a version combination that appeased the constraints so I commented to be able to move on the st2 code base rather than battle with pants plugins.  I eventually installed python3.9 from source on u2204 to be able to run pants 2.17.